site/assets/scss/_content.scss

```diff
@@ -2,19 +2,17 @@
 // Bootstrap docs content theming
 //
 
+:root {
+  --bs-heading-color: var(--bs-emphasis-color);
```

Comment:
I think we'd still need to scope this to the headings that are children of the `.bd-content` container as this changes the value for the headings in our Typography page examples: https://deploy-preview-38262--twbs-bootstrap.netlify.app/docs/5.3/content/typography/.

Reply:
Yep you're right, I've forced pushed the new approach by adding some rules in `.bd-masthead` and `.masthead-followup`.

Note: I haven't changed the color in the footer
<img width="824" alt="Screenshot 2023-03-18 at 08 23 14" src="https://user-images.githubusercontent.com/17381666/226091555-7379bde2-a4a0-4ca0-ad46-b87db1bf473d.png">